src/SupervisionSuccess/utils/getMissingFieldsError.js

```diff
@@ -0,0 +1,7 @@
+function getMissingFieldsError(fields) {
```

Comment:
Small request: can we update this to take in a `state` argument for the state whose row is missing the fields? If `state` is defined, then the error message should end with `... missing for state ${state}.` If `state` is undefined, because it is also missing from the row, then the error message should end with `... missing for a row without a defined state.`

Reply:
Added check for empty string (empty year or state now throws error) and custom error message if state is empty.